library/core/src/array/mod.rs

```diff
@@ -27,6 +28,41 @@ pub(crate) use drain::drain_array_with;
 #[stable(feature = "array_value_iter", since = "1.51.0")]
 pub use iter::IntoIter;
 
+/// Creates an array of type `[T; N]` by repeatedly cloning a value.
+///
+/// The value will be used as the last element of the resulting array, so it
```

Comment:
Should we promise into which slot it goes? Cloned elements should be identical and code relying on the order would be sus imo.

Reply:
I'll remove that hint then.